src/views/guestLayout.js

```diff
@@ -61,6 +62,7 @@ const guestLayout = data => `
         <img src='/images/moz-logo-1color-white-rgb-01.svg' width='100' height='29' loading='lazy' alt='${getMessage('mozilla')}'>
       </a>
       <menu>
+        <li><a href='/breaches'>${getMessage('footer-nav-all-breaches')}</a></li>
         <li><a href='https://support.mozilla.org/kb/firefox-monitor-faq' target='_blank'>FAQ</a></li>
```

Comment:
Actually, just noticed this while grepping for a different PR on `utm_*` parameters:
```sh
git grep -n "https://support.mozilla.org/kb/firefox-monitor-faq" src | cat

src/views/emails/email-2022.js:10:  faq: 'https://support.mozilla.org/kb/firefox-monitor-faq',
src/views/guestLayout.js:76:        <li><a href='https://support.mozilla.org/kb/firefox-monitor-faq' target='_blank'>FAQ</a></li>
src/views/mainLayout.js:105:        <li><a href='https://support.mozilla.org/kb/firefox-monitor-faq' target='_blank'>FAQ</a></li>
src/views/partials/landing.js:70:    <a href='https://support.mozilla.org/kb/firefox-monitor-faq' target='_blank'>${getMessage('see-all-faq')}</a>
```

Where `see-all-faq` string is: 
```sh
git grep -n "see-all-faq" locales/en/ | cat

locales/en/landing.ftl:40:see-all-faq = See all FAQs
```

So looks like "FAQ" is hardcoded 2&times; and uses localized "See all FAQs" elsewhere.
Not necessarily a bug, but just something to confirm possibly.

But maybe that makes sense since they are used in the landing page vs footer:
![Screenshot 2023-03-23 at 4 17 10 PM](https://user-images.githubusercontent.com/557895/227385377-fb530e34-6e63-4933-bb87-b26fc6c49917.png)

Reply:
Oh hm good catch, I'm not sure if we generally translate FAQ but I think it's a reasonable thing to do? Since this landed in 54ad4b6d1 and not the current PR would you mind filing a ticket for this? Thanks!

Reply:
Done-ish. Filed as #2933. /cc @flodolo 

Since it doesn't look like y'all use GitHub issues very much, I'll copy/pasta it to Jira and update this comment with the Jira bug number since that seems like the source of truth(tm).

Reply:
(it should be translated, yes)